sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -14,6 +14,8 @@
 from sklearn.preprocessing import OrdinalEncoder
 
 
+# TODO(1.4): Remove filterwarning when `sparse` is replaced by `sparse_output`.
```

Comment:
Please replace all occurrences of `sparse` by `sparse_output` and keep the filterwarnings only if needed.

Reply:
Done :+1: